packages/swapper/src/swappers/PortalsSwapper/getPortalsTradeQuote/getPortalsTradeQuote.ts

```diff
@@ -217,9 +217,9 @@ export async function getPortalsTradeQuote(
             networkFeeCryptoBaseUnit,
             // Protocol fees are always denominated in buy asset here, this is the downside on the swap
```

Comment:
can probably remove comment, or correct the side if you still find it useful

Reply:
Fixed comment in https://github.com/shapeshift/web/pull/7661/commits/cf10c538270ada3ce15cae3714d21a3a6ebbdc6e and will merge.